doc/guides/internal/index.md

```diff
@@ -0,0 +1,17 @@
+# we assume no responsibility
```

Comment:
?????????

Reply:
what should I put as a disclaimer?

Reply:
> what should I put as a disclaimer?

 Why does this file need a disclaimer?

Reply:
https://github.com/nodejs/node/pull/26665#issuecomment-476299423

Reply:
That's requesting a disclaimer along the lines of "These flags are for Node.js core development usage only. Do not use these flags in your own applications. These flags are not subjected to semantic versioning rules. The core developers may remove these flags in any version of Node.js."

Reply:
(In other words, not a legal disclaimer of no liability, but a disclaimer to not use these flags if you're doing it for internal Node.js core development.)